tools/deploy_wiremock.py

```diff
@@ -0,0 +1,33 @@
+import utils
+import deployment_options
+import pvc_size_utils
+
+
+log = utils.get_logger('deploy_wiremock')
+
+
+def main():
+    deploy_options = deployment_options.load_deployment_options()
+
+    log.info('Starting wiremock deployment')
+
+    utils.set_profile(deploy_options.target, deploy_options.profile)
+
+    deploy_wiremock(deploy_options)
+
+    log.info('Completed to wiremock deployment')
+
+
+def deploy_wiremock(deploy_options):
```

Comment:
ditto (blank lines)

Reply:
yeah, seems it's a convention in the deploy files to keep two blank lines between funcs.